packages/inventory-general-info/src/ConfigurationCard/ConfigurationCard.test.js

```diff
@@ -112,7 +112,8 @@ describe('ConfigurationCard', () => {
     it('should render extra', () => {
         const store = mockStore(initialState);
         const wrapper = render(<ConfigurationCard store={ store } extra={[
-            { title: 'something', value: 'test' }
+            { title: 'something', value: 'test' },
+            { title: 'with click', value: '1 tests', onClick: () => [ 'Something', {}, 'small' ] }
```

Comment:
I think it's very confusing to call it `onClick` when it's a function that returns an array - onClick should be just a function where consumers can do anything what they want to do.

I have two suggestions:

1. rename it to `items` (or whatever) and change it to just an `array`

2. provide handleClick to be consumed `onClick: (e, handleClick) => handleClick([ 'Something', {}, 'small' ])`

Reply:
I like the second approach!